cmake/configs/posix_rpi_common.cmake

```diff
@@ -8,6 +8,7 @@ include(posix/px4_impl_posix)
 add_definitions(
 	-D__PX4_POSIX_RPI
 	-D__DF_LINUX # For DriverFramework
+	-D__DF_RPI # For DriverFramework
```

Comment:
Where exactly? I see some files in https://github.com/PX4/DriverFramework/tree/master/cmake/toolchains which are not used anywhere in the project.